grades1.py

```diff
@@ -0,0 +1,47 @@
+#This code will generate a list that iterates 4 lists containing a student's grade
+#
+#
+#Define main function
+test= .30
+project= .30
+lab= .20
+hw= .20
+
+def main():
+    testGrades= ["tests", 100, 85, 78, 0]
```

Comment:
Pull out of the main function. Make it external input (read from file or from command line).